drivers/clock_control/clock_stm32l4_l5_wb_wl.c

```diff
@@ -26,12 +25,35 @@
 #define z_pllr(v) LL_RCC_PLLR_DIV_ ## v
 #define pllr(v) z_pllr(v)
 
+/**
+ * @brief Return PLL source
+ */
+__unused
+static uint32_t get_pll_source(void)
+{
+	/* Configure PLL source */
+	if (IS_ENABLED(STM32_PLL_SRC_HSI)) {
+		return LL_RCC_PLLSOURCE_HSI;
+	} else if (IS_ENABLED(STM32_PLL_SRC_HSE)) {
+		return LL_RCC_PLLSOURCE_HSE;
+	} else if (IS_ENABLED(STM32_PLL_SRC_MSI)) {
+		return LL_RCC_PLLSOURCE_MSI;
+	}
+
+	return -ENOTSUP;
```

Comment:
it is not correct to return `-ENOTSUP` as return type is `uint32_t`.
Maybe `#ifdef` and `#error` combination is better?